merge_outputs.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+NTUPLETAG=$1
+CHANNEL=$2
+ERA=$3
+
+case ${CHANNEL} in
+    "electron")
+        scope=ee
+        sample="Electron"
+        ;;
+
+    "muon")
+        scope=mm
+        sample="Muon"
+        ;;
+
+    *)
+        echo "[ERROR] Given channel ${CHANNEL} not implemented"
+        exit 42
+esac
+
+
+basedir=/store/user/mburkart/CROWN/ntuples/${NTUPLETAG}/CROWNRun/${ERA}/
```

Comment:
change these to user instead of mburkart

Reply:
Fixed